src/BTVNanoCommissioning/workflows/ctag_Wc_valid_sf.py

```diff
@@ -518,7 +513,7 @@ def process_shift(self, events, shift_name):
                 out_branch,
                 np.where(
                     (out_branch == "SoftMuon")
-                    | (out_branch == "MuonJet")
+                    # | (out_branch == "MuonJet")
```

Comment:
Did I understand correctly you want to store PFcands in the `MuonJet` collection so you commented out?(also `OtherJets` also not included)